style.css

```diff
@@ -558,19 +605,23 @@ textarea:hover::placeholder {
   #contact-form {
     display: flex;
     flex-direction: row;
-    align-items: flex-start;
+    align-items: center;
+    justify-content: center;
     margin-top: 5%;
     padding-left: 0.938rem;
     padding-right: 0.938rem;
     margin-left: auto;
     margin-right: auto;
+    max-width: 1080px;
   }
 
   #contact-form-title {
     display: flex;
     width: 40%;
-    padding-left: 10%;
-    padding-bottom: 0%;
+    padding: 0%;
+    padding-bottom: 10%;
+    padding-left: 5%;
+    font-size: 1.75rem;
   }
```

Comment:
- [ ] [OPTIONAL] Please ensure that this text covers at least 32% of the all desktop screen sizes. This is to ensure your UI is improved and your design matches the Figma design ☺️.

Your design 😎 
![image](https://github.com/danielf-canon/Portfolio/assets/99394374/dac19c48-2228-4d51-bb8b-8c4e6a8a6100)